integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/gcp/GcpUtil.java

```diff
@@ -44,15 +45,15 @@ public void stopHostGroupInstances(List<String> instanceIds) {
     }
 
     public void cloudStorageListContainer(String baseLocation, String selectedObject, boolean zeroContent) {
-        listSelectedObject(baseLocation + selectedObject, zeroContent);
+        listSelectedObject(StringUtils.join(List.of(baseLocation, selectedObject), "/"), zeroContent);
```

Comment:
What's wrong with the simple + operator?

Reply:
The `/` was missing and this generates `NotFounException` at 
```
2022-04-04 18:11:09,981 [TestNG-test=sdx_backup_restore_e2e_tests-1] INFO  c.s.i.c.u.g.action.GcpClientActions [SdxBackupRestoreTest.testSDXBackupRestoreCanBeSuccessful] -  Google GCS Bucket: cloudbreak-dev
2022-04-04 18:11:09,981 [TestNG-test=sdx_backup_restore_e2e_tests-1] INFO  c.s.i.c.u.g.action.GcpClientActions [SdxBackupRestoreTest.testSDXBackupRestoreCanBeSuccessful] -  Google GCS Key Prefix: apitestaf69f5e231c6477fa1ab1f129ce0d527
2022-04-04 18:11:09,981 [TestNG-test=sdx_backup_restore_e2e_tests-1] INFO  c.s.i.c.u.g.action.GcpClientActions [SdxBackupRestoreTest.testSDXBackupRestoreCanBeSuccessful] -  Google GCS Object: /apitestaf69f5e231c6477fa1ab1f129ce0d527backups
2022-04-04 18:11:12,070 [TestNG-test=sdx_backup_restore_e2e_tests-1] INFO  c.s.i.c.u.g.action.GcpClientActions [SdxBackupRestoreTest.testSDXBackupRestoreCanBeSuccessful] -  Google GCS object: javax.ws.rs.NotFoundException: HTTP 404 Not Found...
```

Reply:
I was referring to how the string concatenation happens:
`StringUtils.join(List.of(...`
it looks complicated and see no benefit to just simply `+` operator